lib/internal/abort_controller.js

```diff
@@ -372,18 +374,50 @@ ObjectDefineProperty(AbortSignal.prototype, SymbolToStringTag, {
 
 defineEventHandler(AbortSignal.prototype, 'abort');
 
+// https://dom.spec.whatwg.org/#dom-abortsignal-abort
 function abortSignal(signal, reason) {
+  // 1. If signal is aborted, then return.
   if (signal[kAborted]) return;
+
+  // 2. Set signal's abort reason to reason if it is given;
+  //    otherwise to a new "AbortError" DOMException.
   signal[kAborted] = true;
   signal[kReason] = reason;
+  // 3. Let dependentSignalsToAbort be a new list.
+  const dependentSignalsToAbort = [];
+  // 4. For each dependentSignal of signal's dependent signals:
+  if (signal[kDependantSignals]) {
+    SetPrototypeForEach(signal[kDependantSignals], (s) => {
```

Comment:
nit: I've always been a bit unhappy with the naming for this... Would be please consider adding a comment here indicating that `SetPrototypeForEach` is `Set.prototype.forEach` as opposed to "setting the prototype for each ..."

Reply:
Haha, indeed. I'll add a comment